AI/finder/financial_eval.py

```
@@ -0,0 +1,229 @@
+            elif roe >= 0: roe_score = 2
+            else: roe_score = 1
+        else:
+            roe_score = 3
+
+        # 4. 매출 성장률 (Revenue Growth: 최근 2년)
+        if len(g) >= 2:
+            rev_growth = (g.iloc[-1]["revenue"] - g.iloc[-2]["revenue"]) / g.iloc[-2]["revenue"] if g.iloc[-2]["revenue"] else None
+        else:
+            rev_growth = None
+        if rev_growth is not None:
+            if rev_growth >= 0.10: rev_score = 5
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)